config/releases.exs

```diff
@@ -27,7 +27,14 @@ config :lenra,
   gitlab_ci_ref: "master",
   template_url: System.fetch_env!("TEMPLATE_URL"),
   lenra_email: System.fetch_env!("LENRA_EMAIL"),
-  lenra_app_url: System.fetch_env!("LENRA_APP_URL")
+  lenra_app_url: System.fetch_env!("LENRA_APP_URL"),
+  pipeline_runner: System.get_env("PIPELINE_RUNNER", "gitlab"),
+  kubernetes_api_url: System.get_env("KUBERNETES_API_URL") || "https://#{System.get_env("KUBERNETES_SERVICE_HOST")}",
```

Comment:
In the case KUBERNETES_API_URL & KUBERNETES_SERVICE_HOST are not set you will have an url: `"https://"`. Maybe only let KUBERNETES_API_URL with `fetch_env!`